src/chttpd_db.erl

```diff
@@ -93,7 +93,7 @@ handle_changes_req1(#httpd{}=Req, Db) ->
             couch_stats:decrement_counter([couchdb, httpd, clients_requesting_changes])
         end;
     _ ->
-        Msg = <<"Supported `feed` types: normal, continuous, longpoll, eventsource">>,
+        Msg = <<"Supported `feed` types: normal, continuous/stream, longpoll, eventsource">>,
```

Comment:
in general a person can misread every information you present and in this case i would doubt that this happens often

how would you present the message?